tools/theme/components/badge.ts

```diff
@@ -0,0 +1,47 @@
+import { ComponentSingleStyleConfig } from '@chakra-ui/react'; // Single component style config
+
+// Badge style config
+const Badge: ComponentSingleStyleConfig = {
+  // Base style
+  baseStyle: {
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: '23px',
+    textTransform: 'inherit',
+    fontWeight: 'normal',
+    fontSize: '16px',
+    height: '32px',
+    paddingX: '1rem',
+    paddingY: '4px',
+  },
+
+  // Variants (completed, inProgress, pending, rejected, expiring, expired, active, inactive)
+  variants: {
+    completed: {
+      bg: 'badge.completed',
+    },
+    inProgress: {
+      bg: 'badge.inProgress',
+    },
+    pending: {
+      bg: 'badge.pending',
+    },
+    rejected: {
+      bg: 'badge.rejected',
+    },
+    expiring: {
+      bg: 'badge.expiring',
+    },
+    expired: {
+      bg: 'badge.rejected',
```

Comment:
Should this be expired?

Reply:
The expired color and rejected color are the same so it's being reused here. If that's confusing I could rename the color with something that tries to encompass both rejected and expired, or restate the color again in `tools/theme/colors.ts` with the name expired

Reply:
I think to reduce confusion, we should just have the two colours in `colors.ts`. There will be a bit of repeated "code", but it is clearer and reduces potential confusion.